aip/general/0151.md

```diff
@@ -130,7 +147,9 @@ is a breaking change.
 
 ## Changelog
 
-- **2022-06-??**: Added compatibility section
+- **2024-04-??**: Provided pattern for validation on RPCs returning
+  long-running operationss
```

Comment:
typo: operations

also this and below sentences should be terminated by a dot.

Reply:
Done.